README.md

```diff
@@ -302,23 +412,57 @@ client = Client(key=NEXMO_API_KEY, secret=NEXMO_API_SECRET)
 verify = Verify(client)
 ```
 
+​
+
+## Code snippets
```

Comment:
The README should have just some bare-bones code showing how to invoke ... full blown snippets should be in the code-snippet repos. Having a section of just code snippets means duplicated effort.